packages/itwinui-css/src/date-picker/date-picker.scss

```diff
@@ -143,6 +143,17 @@ $iui-date-picker-today-circle-size: 32px;
   &-today {
     @include iui-calendar-day-today-base;
   }
+
+  &[aria-disabled='true'] {
+    color: var(--iui-color-text-muted);
+    background-color: transparent;
+    font-weight: var(--iui-font-weight-normal);
+    cursor: not-allowed;
+
+    @media (forced-colors: active) {
+      color: GrayText;
+    }
+  }
```

Comment:
Can we use `--iui-color-text-disabled` for the disabled dates and `--iui-color-text-muted` for dates that are before / after the current month?

Reply:
I'm checking other calendars to see what they're doing.

Google Flights doesn't show dates before/after the month, only active & disabled:
![Screenshot 2023-03-15 at 9 34 39 AM](https://user-images.githubusercontent.com/849817/225324655-e3548ab8-b0c4-4894-9d3b-96c2da130816.png)

Same for Expedia:
![Screenshot 2023-03-15 at 9 36 37 AM](https://user-images.githubusercontent.com/849817/225325062-302c5a23-d654-4483-9607-3ad6afbaf0e8.png)

I can only find a week long calendar view with disabled dates on Apple's Genius Bar:
![Screenshot 2023-03-15 at 9 41 24 AM](https://user-images.githubusercontent.com/849817/225326481-fd65193e-3a51-47fa-ad44-f532f1f9b5bf.png)

Continuing to look at others for inspiration.  Would it be considered breaking change to no longer show dates outside of the month & only show active / disabled dates?

Reply:
Not sure if it would necessarily be a breaking change, but I do know that this decision was made deliberately as we wanted to prevent layout shift by showing the same number of rows for every month. I believe the native date input was used as inspiration: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local

Reply:
Firefox differentiates the "disabled" dates with the "outside current month" dates using different opacities:
![](https://user-images.githubusercontent.com/9084735/225330882-5cf9d59d-3ffa-4cec-8b52-d0d45eebbac5.png)

Chrome doesn't make the distinction, but also disables the "previous"/"next" month buttons, which I like so I think I might incorporate that into the design.
![](https://user-images.githubusercontent.com/9084735/225331127-a4343f28-2260-4fca-bd47-6de4c8623025.png)

Reply:
check out the latest version after my recent changes:
https://itwin.github.io/iTwinUI/1135/css/date-picker.html
https://itwin.github.io/iTwinUI/1135/react/?path=/story/core-datepicker--some-dates-disabled

Reply:
[This link here](https://itwin.github.io/iTwinUI/1135/css/date-picker.html), looks like font sizes are different between the 2 samples?
![Screenshot 2023-03-17 at 12 10 01 PM](https://user-images.githubusercontent.com/849817/225958929-2e5574a7-ffd1-4290-b66e-24d304eb344a.png)

Reply:
oh wow lol. good catch, fixed